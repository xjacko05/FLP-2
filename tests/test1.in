A 5
A B
A C
A D
B C
B D
C D
