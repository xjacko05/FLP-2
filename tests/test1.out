A-B B-C C-D D-A 
A-B B-D D-C C-A 
A-C C-B B-D D-A 
